src/sentry/tasks/symbolication.py

```diff
@@ -399,6 +435,7 @@ def symbolicate_event(
     data: Event | None = None,
     queue_switches: int = 0,
     has_attachments: bool = False,
+    symbolicate_platforms: list[SymbolicatorPlatform] = [],
```

Comment:
the proliferation of these task functions is such a huge pain :-(

I might have an idea how to solve / improve this, I might experiment with that later on.